src/libs/migrations/KeyReportActionsByReportActionID.js

```diff
@@ -0,0 +1,59 @@
+import _ from 'underscore';
+import Onyx from 'react-native-onyx';
+import Log from '../Log';
+import ONYXKEYS from '../../ONYXKEYS';
+
+/**
+ * This migration updates reportActions data to be keyed by reportActionID rather than by sequenceNumber.
+ *
+ * @returns {Promise}
+ */
+export default function () {
+    return new Promise((resolve) => {
+        const connectionID = Onyx.connect({
+            key: ONYXKEYS.COLLECTION.REPORT_ACTIONS,
+            waitForCollectionCallback: true,
+            callback: (allReportActions) => {
+                Onyx.disconnect(connectionID);
+
+                if (!allReportActions) {
+                    Log.info('[Migrate Onyx] Skipped migration KeyReportActionsByReportActionID');
```

Comment:
```suggestion
                    Log.info('[Migrate Onyx] Skipped migration KeyReportActionsByReportActionID because there were no report actions');
```

Reply:
done